src/main/java/org/jvnet/hudson/update_center/Main.java

```diff
@@ -227,6 +229,16 @@ private JSONObject buildUpdateCenterJson(MavenRepository repo, LatestLinkBuilder
         return root;
     }
 
+    private JSONArray buildWarnings() {
+        try {
+            String warningsText = IOUtils.toString(this.getClass().getClassLoader().getResourceAsStream("warnings.json"));
+            JSONArray warnings = JSONArray.fromObject(warningsText);
+            return warnings;
+        } catch (IOException e) {
+            return new JSONArray();
```

Comment:
Good point, as I wrote above generation should just fail in this case. I will modify it like that.